src/ClassifierInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Classifier;
+
+/**
+ * `Classifier` is a class finder that represents the classes found.
+ */
+interface ClassifierInterface
```

Comment:
What is this interface about? 
Having only `find` method here makes me confused how it can be used. 
There are some point to rethink the interface

1. There are no `with` methods, so it can't be configured additionally
2. There are no parameters in `find` method
3. It represents not a service, so every time you will have a new instance of something, it's better to pass the implementation
4. `find` return type is `Generator`, so it can't be called twice

What benefits to have the `ClassifierInterface` with one method inside?

Reply:
4. Why?